test/board_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns board-test
+  (:require [clojure.test :refer [deftest testing is]]
+            [board :refer :all]))
+
+(deftest board-test
+  (testing "creates a fresh board"
+    (is (= [1 2 3 4 5 6 7 8 9] (create-board))))
```

Comment:
You are referencing an empty board a few times. Perhaps we can define it once and use it through out the test suite.

Reply:
(create-board) is a fresh board. This first test is simply used to see if (create-board) outputs an empty board correctly